src/supernova/mod.rs

```diff
@@ -181,12 +181,10 @@ pub struct RunningClaim<G1, G2, C1, C2>
 where
   G1: Group<Base = <G2 as Group>::Scalar>,
   G2: Group<Base = <G1 as Group>::Scalar>,
-  C1: EnforcingStepCircuit<G1::Scalar>,
```

Comment:
This shouldn't be deleted.

Reply:
Yup, good catch — restored. I initially thought `C1` would be removed completely and forgot to add this back when it landed in the `PhantomData`.